viadot/utils.py

```diff
@@ -33,8 +35,11 @@ def handle_api_response(
         params (Dict[str, Any], optional): the request params also includes parameters such as the content type. Defaults to None.
         headers: (Dict[str, Any], optional): the request headers. Defaults to None.
         timeout (tuple, optional): the request times out. Defaults to (3.05, 60 * 30).
+        method (Literal ["GET", "POST"], optional): REST API method to use. Defaults to "GET".
+        body (str, optional): Data to send using post method. Defaults to None.
```

Comment:
I think HTTP methods are capitalized by convention, so it should be "using POST method".